dev-tools/trigger-website.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 TOKEN=$1
 MSG_SUFFIX=${2:-''}
 
-if [ ! -z "$MSG_SUFFIX" ]
+if [ -n "$MSG_SUFFIX" ]
```

Comment:
Can we revert this change?

Reply:
Why? I agree that it's not strictly related to this PR, but IMHO the code is more clear...

For example, which of the following PHP code examples do you prefer?
```php
if (!($a === false)) {
```
vs
```php
if ($a !== false)) {
```

Furthermore, it fixes [this warning](https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/568483121#L633)

Reply:
Sorry, I was thinking it's some leftover after old PRs.